src/fides/api/ops/service/connectors/saas/connector_registry_service.py

```diff
@@ -54,7 +54,11 @@ def validate_config(cls, config: str) -> str:
     @validator("dataset")
     def validate_dataset(cls, dataset: str) -> str:
         """Validates the dataset at the given path"""
-        FidesopsDataset(**load_dataset(dataset)[0])
+        try:
```

Comment:
Does it make sense to add this try/except block to the other decorators? My guess is that they also swallow the exceptions. That can definitely be a follow up ticket since it's more to test/QA

Reply:
ah I actually meant to revert this change you'd added - I wasn't seeing the swallowed exceptions on my end:

If I create an error with my datasets:

```bash
datastores specified: postgres
configuring infrastructure for postgres
[+] Running 4/5
 ⠿ Network fides_default               Created                                                                                                                            0.0s
 ⠿ Container fides-redis-1             Healthy                                                                                                                           17.4s
 ⠿ Container fides-fides-db-1          Healthy                                                                                                                           17.4s
 ⠿ Container fides-postgres_example-1  Healthy                                                                                                                           17.4s
 ⠿ Container fides-fides-1             Waiting
```

If I go into the  fides docker container with the try/except removed:
```2022-12-13 19:50:32.640 [INFO] (main:setup_server:228): Validating SaaS connector templates...
2022-12-13 19:50:33.860 [ERROR] (main:setup_server:235): Error occurred during SaaS connector template validation: 1602 validation errors for ConnectorRegistry
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 0 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 1 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 3 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 5 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 7 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 10 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 12 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 14 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 16 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 17 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 19 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 20 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 21 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 22 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 23 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 26 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 27 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 28 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 29 -> fides_meta -> blah
  field required (type=value_error.missing)
__root__ -> adobe_campaign -> dataset -> collections -> 0 -> fields -> 30 -> fides_meta -> blah
  field required (type=value_error.missing)
...
```

Reply:
I'm pretty sure I did but it's possible I wasn't looking in the right place. If it's no longer an issue then I'm okay with it being removed 👍